README.md

```diff
@@ -1,163 +1,257 @@
 ```
 
-Run it with different parameters
+### 11. Update a Pipeline
+
+Pipelines can be modified with new launch parameters using the `update` command. 
 
 ```
-tw launch sleep_one_minute --params=<(echo 'timeout: 30')
+$ tw pipelines update -n sleepy-flow --params=<(echo 'timeout: 30')
 ```
 
-Directly launch a Github pipeline not defined at launchpad
+### 12. Launch a workflow directly
+
+It possible to directly launch Git workflows that have not been added to a workspace as Pipelines.
 
 ```
-tw launch nextflow-io/hello
+$ tw launch nextflow-io/hello
 ```
 
-Update a launchpad pipeline
+### :gift: Bonus: Activate autocomplete 
+
+Install autocomplete in your current session, type `tw pi` then press tab twice.
 
 ```
-tw pipelines update -n sleep_one_minute --params=<(echo 'timeout: 30')
+$source <(tw generate-completion)
 ```
 
-## Launch usage examples
+Launch Examples
+---------------
 
-Run a workspace defined pipeline with custom parameters file:
+The `tw launch` command provides a similar user experience to `nextflow run` with the benefits of using Tower.
+
+### 1. Run a workspace Pipeline with a custom parameters file
 
 ```
 tw launch sarek --params ./myparams.yaml
 ```
 
-Run any Nextflow pipeline using the primary compute environment:
+### 2. Run any Nextflow workflow using the primary Compute Environment
 
 ```
-tw launch nextflow-io/hello 
+$ tw launch nextflow-io/hello 
 ```
 
-Run any Nextflow pipeline setting a profile.
+### 3. Run any Nextflow pipeline setting a profile
 
 ```
-tw launch nf-core/sarek --profile test,docker --params ./myparams.yaml
+$ tw launch nf-core/sarek --profile test,docker --params ./myparams.yaml
 ```
 
-Select the compute environment that you want to use:
+### 4. Select a specific Compute Environment
 
 ```
-tw launch nf-core/sarek --compute-env "aws seqera" --profile test,docker
+$ tw launch nf-core/sarek --compute-env "aws-ce" --profile test,docker
 ```
 
-### Development
+
+Build binary development version
+--------------------------------
+
+The Tower CLI is based on a Java native compilation, Micronaut HTTP declarative client and Tower domain classes.
+
+1. Download GraalVM (Java 11 version) from [this link](https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.2.0).
+
+2. Install `native-image` tool:
+
+    ```
+    $ gu install native-image
+    ``` 
+
+3. Create the native client:
+
+    ```
+    $ ./gradlew nativeImage
+    ```
+
+4. then run
+
+    ```
+    $ ./build/graal/tw
+    ```
+
+Development
+-----------
 
 To force reload SNAPSHOT dependencies run:
 
 ```
 ./gradlew clean build --refresh-dependencies
 ```
 
+Using non-binary development versions
+-------------------------------------
+
+You can run a non-binary development version running the `./tw` script in the root of this repository.
+
+License
+-------
+
+[Mozilla Public License v2.0](https://github.com/seqeralabs/tower-cli/blob/master/LICENSE.txt)
+
 ### Credits & Links
```

Comment:
Not sure if we want to keep this links, I think I'd completely remove this section. We only used them for the first prototype. In fact some of them are a bit outdated and we are not using what they mention, at next CLI version we're using the official GraalVM gradle pluggin to build native binaries.